pontoon/base/templatetags/helpers.py

```diff
@@ -1,8 +1,9 @@
 import cgi
 import datetime
 import json
-import urllib
-import urlparse
+
+from six import text_type
+from six.moves.urllib import parse as urlparse
```

Comment:
urlparse module is removed in python3 but yes, it can mislead contributors. Used six_parse as you suggested before.